Utilities/SharedMutex.h

```diff
@@ -14,7 +15,7 @@
 //! std::shared_mutex is not available until C++17.
 class shared_mutex final
```

Comment:
Btw, `shared_mutex` (untimed) is now available with VS2015 Update 2. Have you tried it?